go.mod

```diff
@@ -327,5 +327,5 @@ replace (
 	github.com/filecoin-project/dagstore => github.com/celestiaorg/dagstore v0.0.0-20221014072825-395797efb659
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/libp2p/go-libp2p-pubsub v0.7.0 => github.com/celestiaorg/go-libp2p-pubsub v0.6.2-0.20220812132010-46b2a019f2f2
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.5.0-tm-v0.34.20
+	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.5.0-tm-v0.34.20.0.20221214133108-8692e50bffa7
```

Comment:
do we want a special tag for this? it should be fixed in the new release, but we already released v1.6, so we'd have to have a weird release to continue semi following semver. I'm also ok with completely not following semver for core, and releasing v1.5.1, but would prefer not to release 1.12 that doesn't have the many breaking changes in v1.11

Reply:
I'm okay with not having a special tag in favour if semver for core.

Reply:
Ok, from me as well. Next bump should have a real version please